job/server/src/main/java/alluxio/job/plan/transform/format/csv/Decimal.java

```diff
@@ -50,11 +51,23 @@ public int getScale() {
     return mScale;
   }
 
+  /**
+   * Returns the decimal's precision and scale from the type definition.
+   *
+   * @param type the type definition, like "decimal(10, 2)"
+   * @return the decimal's precision and scale as a Pair
+   */
+  public static Pair<Integer, Integer> getPrecisionAndScale(String type) {
```

Comment:
are there some unit tests for this? Will this handle extraneous whitespaces well?

Reply:
This is not actually new code but I'll add some test since this class doesn't seem to have a unit test at all.

Reply:
Yes, I know it is old code, but thanks for the test cases!